transcom/mymove
# MB 16329 playwright testing for too approving delivery address change for hhg

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16329)

## Summary
 This PR creates a new Playwright test for the TOO approving a delivery address change request for an HHG shipment.

The test should perform the following steps:
-  Logs in as a TOO user
- Navigates to a move with approvals requested
-  Checks to make sure "review required" alert is on the Move Details page
- Navigates to edit shipment details and check for "Request needs review..." alert on the top of page
- Checks for the "Pending delivery location..." alert under delivery location section
- Clicks Review request, and fills out modal with "yes" radio button and leaves a comment
- Saves changes in the modal, and checks the "Changes sent to contractor" alert appears at the top of page
- Presses save on edit shipment details page, and displays updated destination address on move details page.
- Navigates back to the move queue, and enters the same move code used previously
- Checks the status is approved


## Verification Steps for the Author & Reviewers:

1. Check out branch: MB-16329-playwright-testing-for-too-approves-delivery-address-change-for-hhg
2. Run  `npx playwright test playwright/tests/office/txo/tooFlows.spec.js --trace="on"`
3. Look for the TOO user approves a delivery address change request for an HHG shipment test in the playback trace
4. Watch the playback trace and determine if the integration test follows the AC [here](https://dp3.atlassian.net/browse/MB-16329).
